src/PowerShellEditorServices/Session/PowerShell5Operations.cs

```diff
@@ -3,30 +3,98 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Host;
 using System.Management.Automation.Runspaces;
 
 namespace Microsoft.PowerShell.EditorServices.Session
 {
-    internal class PowerShell5Operations : PowerShell4Operations
+    internal class PowerShell5Operations : IVersionSpecificOperations
```

Comment:
Ideally we can dispense with this whole interface, but perhaps that is too big a change for this PR?

Reply:
I say get rid of it.  I'd say keep it in case we want to support PSCore only features, but we should probably come up with a better solution anyway.

Reply:
I agree with Patrick

Reply:
I'll leave this for a later cleanup PR

Reply:
Ya, this PR is huge enough already.  :-)